<launch>

    <node pkg="icp_registration" type="icp_node" name="icp_node" output="screen">
        <rosparam file="$(find icp_registration)/config/icp.yaml" command="load"/>

    </node>
    
</launch>